# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Free Software Foundation, Inc.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2002-01-06 00:19+ZONE\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: ENCODING\n"

#: .././Pyblio/Autoload.py:57
msgid "warning: can't import %s: %s"
msgstr ""

#: .././Pyblio/Config.py:53
msgid "value of `%s' should be of type %s"
msgstr ""

#: .././Pyblio/Config.py:196
msgid "String"
msgstr ""

#: .././Pyblio/Config.py:205
msgid "Boolean"
msgstr ""

#: .././Pyblio/Config.py:224
msgid "Integer"
msgstr ""

#: .././Pyblio/Config.py:226
msgid "Integer under %d"
msgstr ""

#: .././Pyblio/Config.py:228
msgid "Integer over %d"
msgstr ""

#: .././Pyblio/Config.py:230
msgid "Integer between %d and %d"
msgstr ""

#: .././Pyblio/Config.py:242
msgid "Element in `%s'"
msgstr ""

#: .././Pyblio/Config.py:262
msgid "Tuple (%s)"
msgstr ""

#: .././Pyblio/Config.py:283
msgid "List (%s)"
msgstr ""

#: .././Pyblio/Config.py:308
msgid "Dictionary (%s, %s)"
msgstr ""

#: .././Pyblio/Fields.py:265
msgid "Illegal year value"
msgstr ""

#: .././Pyblio/Fields.py:269
msgid "Illegal month value"
msgstr ""

#: .././Pyblio/Fields.py:273
msgid "Illegal day value"
msgstr ""

#: .././Pyblio/Fields.py:443
msgid "in %s"
msgstr ""

#: .././Pyblio/LyX.py:39
msgid "no input pipe `%s'"
msgstr ""

#: .././Pyblio/LyX.py:45
msgid "no output pipe `%s'"
msgstr ""

#: .././Pyblio/Open.py:97
msgid "method `%s' provides no opener"
msgstr ""

#: .././Pyblio/Open.py:105 :151
msgid "File `%s' does not exist"
msgstr ""

#: .././Pyblio/Open.py:116 :162
msgid "don't know how to open `%s'"
msgstr ""

#: .././Pyblio/Open.py:143
msgid "method `%s' provides no iterator"
msgstr ""

#: .././Pyblio/Format/BibTeX.py:350 :408
msgid "key `%s' is malformed"
msgstr ""

#: .././Pyblio/Format/BibTeX.py:371 :392
msgid "%s:%d: key `%s' already defined"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:38
msgid "Last Name, First Name"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:39
msgid "Text"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:42
msgid "Reference"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:258
msgid "Day"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:267
msgid "Month"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:276
msgid "Year"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:301
msgid "Invalid day field in date"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:309
msgid "Invalid month field in date"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:317
msgid "Invalid year field in date"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:426
msgid "Entry type"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:428 .././Pyblio/GnomeUI/Config.py:482
msgid "Key"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:471
msgid "Create Field"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:539 .././Pyblio/GnomeUI/Fields.py:191
msgid "Mandatory"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:539 .././Pyblio/GnomeUI/Fields.py:192
msgid "Optional"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:539
msgid "Extra"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:601
msgid "Invalid key format"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:609
msgid "Key `%s' already exists"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:676
msgid "Error in native string parsing"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:689
msgid "Edit entry"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:703 :744
msgid "Native Editing"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:756
msgid "Standard Editing"
msgstr ""

#: .././Pyblio/GnomeUI/FileSelector.py:41 .././Pyblio/GnomeUI/Document.py:216
#: :230
msgid "File"
msgstr ""

#: .././Pyblio/GnomeUI/FileSelector.py:74
msgid "Bibliography type:"
msgstr ""

#: .././Pyblio/GnomeUI/Index.py:68
msgid "Add..."
msgstr ""

#: .././Pyblio/GnomeUI/Index.py:69
msgid "Edit..."
msgstr ""

#: .././Pyblio/GnomeUI/Index.py:70 .././Pyblio/GnomeUI/Search.py:178
msgid "Delete"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:109
msgid "Search"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:133 .././Pyblio/GnomeUI/Fields.py:225
msgid "Field"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:134
msgid "Pattern"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:151
msgid "Simple Search"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:155
msgid "Search command:"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:161
msgid "Expert Search"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:199
msgid "Full database"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:241
msgid "internal error during evaluation"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:290
msgid ""
"while compiling %s\n"
"error: %s"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:49
msgid "Select sort criterions"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:60
msgid "Sort direction"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:60
msgid "Sort criterions"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:73
msgid "[Entry Type]"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:74
msgid "[Key Value]"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:99
msgid "Reorder"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:100
msgid "Groups the active fields on the top of the list"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:104
msgid "Unselect all"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:105
msgid "Removes all the sort criterions"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:109
msgid "Set as default"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:110
msgid "Use this sort criterion as the default for new files"
msgstr ""

#: .././Pyblio/GnomeUI/Utils.py:163
msgid "The following errors occured:\n\n"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:63
msgid "_New"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:65
msgid "_Merge with..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:66
msgid "Medline Query..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:70
msgid "_Previous Documents"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:77
msgid "About..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:78
msgid "Submit a Bug Report"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:90
msgid "_Add..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:91
msgid "_Edit..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:92
msgid "_Delete"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:95
msgid "S_ort..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:99 :113
msgid "_Cite"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:100
msgid "_Format..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:104
msgid "_Fields..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:105
msgid "_Entries..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:111
msgid "_File"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:112
msgid "_Edit"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:114
msgid "_Settings"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:115
msgid "_Help"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:119
msgid "Open"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:120
msgid "Save"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:122 .././Pyblio/GnomeUI/Config.py:390
msgid "Add"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:124
msgid "Find"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:125
msgid "Cite"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:127
msgid "Close"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:216 :230
msgid "Previous Documents"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:252
msgid ""
"can't open file `%s' for writing:\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:273
msgid ""
"Error while parsing `%s':\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:292
msgid "New database"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:300 :304
msgid "[no entry]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:301
msgid "[1 entry]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:302
msgid "[%d entries]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:305
msgid "[%d/1 entry]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:306
msgid "[%d/%d entries]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:311
msgid "[modified]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:321
msgid ""
"The database has been modified.\n"
"Discard changes ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:341
msgid "Enter your Medline query"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:347
msgid "Search string"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:351
msgid "Maximum number of results"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:371
msgid "Merge file"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:383 :443
msgid "Open error"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:407
msgid "Merge status"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:418
msgid "Open file"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:470
msgid ""
"The database has been externally modified.\n"
"Overwrite changes ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:480
msgid ""
"Unable to save `%s':\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:488
msgid ""
"An internal error occured during saving\n"
"Try to Save As..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:502
msgid "Save As..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:508
msgid ""
"The file `%s' already exists.\n"
"Overwrite it ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:515
msgid ""
"During opening:\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:533
msgid "Reopen error"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:574
msgid ""
"An entry called `%s' already exists.\n"
"Rename and add it anyway ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:611
msgid "Really remove all the entries ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:631
msgid "Create new entry"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:645
msgid "Really edit %d entries ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:683
msgid "Remove all the %d entries ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:685
msgid "Remove entry `%s' ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:755 :762
msgid ""
"Can't connect to LyX:\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:840
msgid ""
"Please install bug-buddy\n"
"to use this feature"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:851
msgid "This program is copyrighted under the GNU GPL"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:853
msgid "Gnome interface to the Pybliographer system."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:858
msgid "Pybliographer Home Page"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:37
msgid "Format entries"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:51
msgid "Bibliography style:"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:52
msgid "Select style"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:58
msgid "Output format:"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:75
msgid "Output File:"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:76
msgid "Select output file"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:45
msgid "Choose your preferences"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:93
msgid ""
"Item `%s':\n"
"\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:128 .././Pyblio/GnomeUI/Fields.py:157 :339
msgid ""
"Some changes require to restart Pybliographic\n"
"to be correctly taken into account"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:249
msgid "True"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:250
msgid "False"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:393
msgid "Update"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:396 :494 .././Pyblio/GnomeUI/Fields.py:96 :239
msgid "Remove"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:482
msgid "Value"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:491 .././Pyblio/GnomeUI/Fields.py:93 :236
msgid "Set"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:507
msgid "Key:"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:509
msgid "Value:"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:45
msgid "Fields configuration"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:59
msgid "Name"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:59
msgid "Type"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:70
msgid "Name:"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:72
msgid "Type:"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:198
msgid "Entries configuration"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:209
msgid "Entry"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:220
msgid "Entry Name:"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:225
msgid "Status"
msgstr ""

#: .././pybliographer.py:48
msgid "This is %s, version %s"
msgstr ""

#: .././pybliographer.py:52 :59
msgid "This is free software with ABSOLUTELY NO WARRANTY."
msgstr ""

#: .././pybliographer.py:53
msgid "For details, type `warranty'."
msgstr ""

#: .././pybliographer.py:116 :168
msgid "%s: error: can't open file `%s'"
msgstr ""

#: .././pybliographer.py:129
msgid "For help, run %s and type `help' at the prompt"
msgstr ""

#: .././pybliographer.py:138
msgid ""
"Useful commands:\n"
"\thelp     to get some help\n"
"\tquit     to quit\n"
msgstr ""

#: .././scripts/pybcompact.py:29
msgid "usage: pybcompact <latexfile> <bibtexfiles...>"
msgstr ""

#: .././scripts/pybcompact.py:33
msgid "pybcompact: error: %s\n"
msgstr ""

#: .././scripts/pybcompact.py:96
msgid "no entry"
msgstr ""

#: .././scripts/pybcompact.py:141
msgid "can't find the following entries: %s"
msgstr ""

#: .././scripts/pybconvert.py:28
msgid "usage: pybconvert <source>..<target> <input> [output]"
msgstr ""

#: .././scripts/pybconvert.py:37
msgid "pybconvert: bad conversion format"
msgstr ""

#: .././scripts/pybformat.py:33
msgid "usage: pybformat [bibtexfiles...]\n"
msgstr ""

#: .././scripts/pybformat.py:37
msgid "pybformat: error: %s\n"
msgstr ""

#: .././scripts/pybformat.py:43
msgid "pybformat: warning: %s\n"
msgstr ""

#: .././scripts/pybformat.py:140 .././scripts/pybtex.py:85
msgid "can't find style `%s'"
msgstr ""

#: .././scripts/pybtex.py:33
msgid "usage: pybtex <latexfile> [bibtexfiles...]"
msgstr ""

#: .././scripts/pybtex.py:37
msgid "pybtex: error: %s\n"
msgstr ""

#: .././scripts/pybtex.py:58
msgid "pybtex: warning: the following keys were not resolved"
msgstr ""

#: .././scripts/pybtex.py:63
msgid "no style defined"
msgstr ""
